pkg/provider/provider.go

```diff
@@ -60,8 +61,16 @@ func Provider() *schema.Provider {
 				DefaultFunc: schema.EnvDefaultFunc("MZ_SSLMODE", true),
 				Description: "For testing purposes, disable SSL.",
 			},
+			"endpoint": {
```

Comment:
Is the admin endpoint unique to each organization's Materialize account?

Reply:
No, but you'll need some way to override it for tests. Production and staging have different admin endpoints, for example.